packages/dataviews/src/components/dataviews/stories/index.story.tsx

```diff
@@ -536,3 +550,43 @@ export const InfiniteScroll = () => {
 		</>
 	);
 };
+
+export const GridLayoutHiddenTitle = () => {
```

Comment:
I wonder if we need a separate story, or if it can be an arg on the existing default story 🤔 

There are quite a lot of dataviews stories now.

Reply:
Good points! I'll leave it in for now while I'm testing, but happy to remove it 👍

Reply:
Removed in https://github.com/WordPress/gutenberg/pull/71369/commits/bd4b9f44a6c872daf6b9192d4bc3f8e721eb6c54